### ISSUE TITLE
Right Click Scroll

### ISSUE THREAD

**Issue by andyjf0** [potmdehex/homm3tools#48]
Please add scrolling map by holding right click.
It was added to HoMM3 HD mod last year and would be greatly appreciated here too!

Love your work
Peace out

**Comment by tazhel** [comment]
I agree, I think he tried to add it in, because middle mouse click can scroll the map, but a bug makes it jump down before scrolling so it doesn't work correctly. But you can hold middle button now and drag arround to see the function.
